hw/top_earlgrey/ip/alert_handler/top_earlgrey_alert_handler_reg.core

```diff
@@ -0,0 +1,21 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:systems:top_earlgrey_alert_handler_reg:0.1"
```

Comment:
This is timid thing, but I think the naming is better to be `lowrisc:earlgrey:alert_handler_reg:0.1` or `lowrisc:top_earlgrey:same..`

Reply:
I think this is a good idea! I will update these names.